tools/MINGW-packages/mingw-w64-openexr/PKGBUILD

```diff
@@ -27,8 +27,9 @@ replaces=(
 )
 source=("https://github.com/openexr/openexr/archive/v${pkgver}/${_realname}-${pkgver}.tar.gz"
         0006-cmake-soversion.patch)
-sha256sums=('3ff47111ef7e5da6f69330e66e1e90ae620b79df1cedf2512bb9bffe86c2c617'
-            'e65852ac2e5545472ad90830c97e964aa2c71e1795979a8b9867155d578d45ed')
+sha256sums=('61e175aa2203399fb3c8c2288752fbea3c2637680d50b6e306ea5f8ffdd46a9b'
+            'a5ce463815e8d05c879dbe6a255e9f8ac2697cd3ef4242dac1e5cb331f78ed02'
+            '204777b48cde1c4d390789f10c5e429d5295be701f68a7937a90d0ecf271c048')
```

Comment:
Looks like you forgot to upload 0007-export-TypedAttribute-value.patch and add it to the apply_patch_with_msg variable. Either that or you need to remove this sha256sum so the repo will build properly.

Reply:
Thank you. I'll merge this and update the repository release once the "Build pacman repo" and Tests finish running.